### PR TITLE
New package: swaykbdd-1.0

### PR DESCRIPTION
**Pull request by TinfoilSubmarine**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

Closes https://github.com/void-linux/void-packages/issues/34949, briefly tested several months ago.

### CODE REVIEW

File: srcpkgs/swaykbdd/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+short_desc="Keyboard layout switcher for Sway"
+maintainer="Joel Beckmeyer <joel@beckmeyer.us>"
+license="MIT"
+homepage="https://github.com/artemsen/swaykbdd"
+distfiles="https://github.com/artemsen/swaykbdd/archive/v${version}.tar.gz"
+checksum=9de46256cafb2b002d4632e1a2f4ca2383e69b69de4d19d3d2d6f4825ef9fca2
+
+post_install() {
+	vlicense LICENSE
+	vman swaykbdd.1
```
Comment (paper42) [inline srcpkgs/swaykbdd/template:17]: Isn't this already installed by meson?

File: srcpkgs/swaykbdd/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'swaykbdd'
+pkgname=swaykbdd
+version=1.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config"
+makedepends="json-c-devel"
+short_desc="Keyboard layout switcher for Sway"
```
Comment (paper42) [inline srcpkgs/swaykbdd/template:8]: I think this is better (from #35004).
```suggestion
short_desc="Per-window keyboard layout switcher for Sway"
```

File: srcpkgs/swaykbdd/template (hunk 1 above)
Comment (TinfoilSubmarine) [reply srcpkgs/swaykbdd/template]: Yep, fixed

File: srcpkgs/swaykbdd/template (hunk 2 above)
Comment (TinfoilSubmarine) [reply srcpkgs/swaykbdd/template]: Sure, updated

**Pull request merged**